ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Check for API VIP usage
+  shell: |
+    curl -kL --write-out "%{http_code}\n" --silent --output /dev/null  https://{{ apivip }}:6443/readyz  
```

Comment:
dont use shell use get_uri :)

Reply:
sorry, shell is not the right way to use here, switching to `uri` instead :)